profile/README.md

```diff
@@ -79,10 +77,17 @@ Popular application forks with NeoFS support added:
 
 # NeoGo
 
-Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's
-all there. [ansible-neogo-n3](https://github.com/nspcc-dev/ansible-neogo-n3)
-can help you running it.
+Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's all there.
 
-To learn more about Neo smart contract development in go take a look at [neo-go-sc-wrkshp](https://github.com/nspcc-dev/neo-go-sc-wrkshp).
+To learn more about Neo smart contract development in go take a look at
```

Comment:
This certainly doesn't need to be changed.

Reply:
It was truncated to for best view.

Reply:
It's not related to the change you're actually doing. If it's some whitespace cleanup it should be a separate commit. Or not exist at all.